ostd/src/mm/page_table/node/mod.rs

```diff
@@ -331,7 +331,7 @@ where
         debug_assert!(idx < nr_subpage_per_huge::<C>());
         let ptr = paddr_to_vaddr(self.page.start_paddr()) as *mut E;
         // SAFETY: The index is within the bound and the PTE is plain-old-data.
```

Comment:
SAFETY:
 - The page table node is alive. The index is inside the bound, so the page table entry is valid.
 - All page table entries are aligned and accessed with atomic operations only.